abci/Cargo.toml

```diff
@@ -83,4 +83,4 @@ futures = { version = "0.3.30" }
 tokio = { version = "1.40", features = ["macros", "signal", "time", "io-std"] }
 hex = { version = "0.4.3" }
 lazy_static = { version = "1.4.0" }
-pollster = { version = "0.3.0" }
```

Comment:
double-check if we still go with pollster, and don't replace it with solution implemented in https://github.com/dashpay/platform/pull/2232

Reply:
it's only in tests, so we can merge